src/components/Problems/Problems.tsx

```diff
@@ -282,27 +338,13 @@ export const Problems: FC<ProblemsProps> = ({setPageTitle}) => {
         <Menu semesterList={semesterList} selectedId={problemsId} />
         {problems.map((problem) => (
           <Fragment key={problem.id}>

```

Comment:
taka specialna vec... par faktov:
- kazdy element listu ma mat `key` prop
- `<Fragment>` je len starsi zapis `<>`
- do `<Fragment>` vies narvat `key`, ale do `<>` to nejde
- fragment tu uz nemal zmysel ani predtym, je dobry len na zgroupenie viacerych susednych elementov pod jeden "virtualny" - napriklad aby sa nepridal zbytocny `<div>` do stromu. lenze tu uz pod fragmentom bol jediny `<div>` a teraz je jediny `<Problem>`, takze `key` sa prida k Problemu